docker-entrypoint.sh

```diff
@@ -1,9 +1,16 @@
-#!/bin/sh 
+#!/bin/bash 
```

Comment:
Do we really need bash here? What feature was missing from sh?

Reply:
Bash has better conditional tests implementation. Bash is already part of docker image so no additional installation is required.